portfolio/src/main/webapp/style.css

```diff
@@ -130,4 +131,19 @@ body{
   background-color:white;
 }
 
+/*styling for the top container*/
+.container-pt-3 {
+    padding-top:70px;
+    padding-left:20px;
+    padding-bottom:20px;
+    background-color: white;
+}
+
+/*styling for all the containers except for the top one*/
+.container-fluid {
+    padding-bottom:70px;
+    padding-top:20px;
+    padding-left:20px;
+}
+
```

Comment:
Some lines in this css file have no tabs, some have 2 spaces tabs, and some have 4. Could you make sure they're all 2 spaces?